Classes/Player/PKPlayerSettings.swift

```diff
@@ -172,6 +173,12 @@ enum PlayerSettingsType {
     @objc public var contentRequestAdapter: PKRequestParamsAdapter?
     @objc public var licenseRequestAdapter: PKRequestParamsAdapter?
     
+    @objc public var preventsDisplaySleepDuringVideoPlayback: Bool = true {
```

Comment:
Please add documentation, and write down that it is available only from iOS 12.0